python/whylogs/viz/utils/drift_calculations.py

```diff
@@ -183,9 +198,40 @@ def _get_chi2_p_value(target_view_column, reference_view_column) -> Optional[Col
     return chi2_p_value
 
 
+def _get_hellinger_distance(
+    target_view_column: ColumnProfileView, reference_view_column: ColumnProfileView, nbins=None
+) -> Optional[ColumnDriftStatistic]:
+    if not nbins:
+        nbins = MAX_HIST_BUCKETS
+    target_dist_metric = target_view_column.get_metric("distribution")
+    ref_dist_metric = reference_view_column.get_metric("distribution")
+
+    if target_dist_metric is None or ref_dist_metric is None:
+        warnings.warn("Column must have a Distribution Metric assigned to it.")
+        return None
+
+    target_kll_sketch = target_dist_metric.kll.value
+    ref_kll_sketch = ref_dist_metric.kll.value
+
+    if target_kll_sketch.is_empty() or ref_kll_sketch.is_empty():
+        warnings.warn("Distribution sketch must not be empty.")
+        return None
+
+    start = min([target_kll_sketch.get_min_value(), ref_kll_sketch.get_min_value()])
+    end = max([target_kll_sketch.get_max_value(), ref_kll_sketch.get_max_value()])
+    n = target_kll_sketch.get_n() + ref_kll_sketch.get_n()
+    bins, end = _calculate_bins(end=end, start=start, n=n, avg_per_bucket=HIST_AVG_NUMBER_PER_BUCKET, max_buckets=nbins)
+
+    target_pmf = target_kll_sketch.get_pmf(bins)
+    ref_pmf = ref_kll_sketch.get_pmf(bins)
+
+    distance = euclidean(np.sqrt(target_pmf), np.sqrt(ref_pmf)) / np.sqrt(2)
```

Comment:
Might be worth factoring the hellinger distance calculation out into a separate function for better reusability (signature could be two pmfs rather than ColumnProfileViews), can you also add a comment with reference link to the equation used? e.g. https://en.wikipedia.org/wiki/Hellinger_distance

I like that you explicitly use euclidean here as it makes this easier to understand.

Do you need to add numpy as a dependency of whylogs[viz] with this change?

Reply:
Yes, absolutely, will do that!

oh wow, I haven't realized that numpy is not already a core dependency. If that's the case, then yes, it should be added as a dependency to whylogs[viz]

Reply:
done in the latest commit